schemas/com.vimeo/interactive_hot_spot_click_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo interactive video hot spot click",
```

Comment:
The `interaction` schema says in the description: "Schema for a Vimeo interactive video hotspot/panel click"

That seems like the same thing to me, is there a difference?

Reply:
They are similar but are [two separate](https://developer.vimeo.com/player/sdk/reference#events-for-interactive-videos) Vimeo events. The data that comes with them is very similar though, and it didn't seem to make sense to me to have two events that only differ by a property.